src/Bitmovin/api/model/encodings/muxing/AbstractMuxing.php

```diff
@@ -96,4 +102,19 @@ public function setStreams($streams)
         $this->streams = $streams;
     }
 
+    /**
+     * @return string
+     */
+    public function getStreamConditionsMode(): string
```

Comment:
The API client requires PHP 5.6 or higher. This syntax works with PHP 7 only. Please remove `: string`